tests/loss_test.py

```diff
@@ -92,6 +92,19 @@ def test_binary_logistic_loss(self):
     self._test_binary_loss_function(loss.binary_logistic_loss, sigmoid,
                                     reference_impl)
 
+  def test_binary_sparsemax_loss(self):
+    # @vroulet: There is no reference implementation for this loss as far as I know.
+    # Maybe it would be better to allow no reference implementation in
+    # _test_binary_loss_function
+    def reference_impl(label, logit):
+      scores = jnp.where(label, -logit, logit)
+      return jnp.where(scores < -1.0, 0.0, jnp.where(scores > 1.0, scores, (scores + 1.0)**2/4))
+
+    def invlink(logit):
```

Comment:
We can call it `sparse_sigmoid`. Like the usual sigmoid, we have `sparse_sigmoid(0) = 0.5`.

Reply:
I added it too!